src/templates/html/application/ApplicationHTML.tsx

```diff
@@ -147,7 +142,7 @@ function Boundary(props: { data: PlanXExportData[] }): JSX.Element {
 }
 
 function ProposalDetails(props: {
-  data: PlanXExportData[];
+  data: QuestionAndResponses[];
```

Comment:
This line is a low-key very meaningful type change / `PlanXExportData[]` is a real legacy quirk :raised_hands: :broom: